README.md

```diff
@@ -0,0 +1,47 @@
+# RecycleMe
+
+### Features
+* Improve your skills at recycling! 
+* Track your progress over time!
+
+### Install
+* Clone this repo
+* run ```bundle install```
+* run ```open index.html```
```

Comment:
include a line to create the database: run ` rails db:create`

Reply:
Updated per your suggestions